src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const splittedSource = sourceString.split(';');
```

Comment:
Instead of using the 'split' method and a 'for' loop, consider using more modern iteration methods like 'split' followed by 'filter' and 'forEach' to process the styles. This will make the code more declarative and easier to understand.